internal/repository/index.go

```diff
@@ -361,14 +379,12 @@ func (idx *Index) encode(w io.Writer) error {
 }
 
 // Finalize sets the index to final and writes the JSON serialization to w.
```

Comment:
The second half of this comment no longer applies.

Reply:
is fixed.